scrapers/StateOfTexas.mjs

```diff
@@ -5,28 +5,35 @@ export default ({ database, DataScraper }) => {
 	return DataScraper(database, 'StateOfTexas', async () => {
 		const data = [];
 		const baseUrl = 'https://api.tnris.org/api/v1';
-		const id = '2679b514-bb7b-409f-97f3-ee3879f34448';
-		const collections = `collections/${id}`;
-		const resources = `resources?collection_id=${id}`;
-		const collectionsRequest = await fetch(`${baseUrl}/${collections}`);
-		const collection = await collectionsRequest.json();
-		const resourcesRequest = await fetch(`${baseUrl}/${resources}`);
-		const resource = await resourcesRequest.json();
+		const catalouge = await fetch(`https://api.tnris.org/api/v1/collections_catalog/`);
+		const catalougeData = await catalouge.json()
+		const datasets = catalougeData.results.map(dataset => {
+			return dataset.collection_id
+		})
+		for (let i = 0 ; i < datasets.length; i++){ //forEach loops dont work with async
+			const id = datasets[i]
+			const collections = `collections/${id}`;
+			const resources = `resources?collection_id=${id}`;
+			const collectionsRequest = await fetch(`${baseUrl}/${collections}`);
+			const collection = await collectionsRequest.json();
+			const resourcesRequest = await fetch(`${baseUrl}/${resources}`);
+			const resource = await resourcesRequest.json();
 
-		const { publication_date, description, name } = collection;
-		const { results } = resource;
-		const created = moment(publication_date, 'YYYY-MM-DD').unix();
+			const { publication_date, description, name } = collection;
+			const { results } = resource;
+			const created = moment(publication_date, 'YYYY-MM-DD').unix();
 
-		results.map(item => {
-			return data.push({
-				url: item.resource,
-				updated: 0,
-				created,
-				description,
-				name,
+			results.forEach(item => {
+				console.log(item) //takes a long time so log each dataset to see progress
```

Comment:
Putting console.logs willy nilly in code isn't great, with enough of them it can be hard to sift through logs. Although fine for testing, I want to see this removed when it's been tested. Perhaps we look into adding some log levels?